app/views/shared/_select_input.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class ="form-group row">
+  <%= form.label key, {class:"col-sm-2 col-form-label"} %>
+  <div class="col-sm-10">
+    
+    <%= form.select key, select_list, {}, { class: 'form-control'} %>
+
+  </div> 
+</div>
+
+<% obj.errors.full_messages_for(key).each do |message| %>
+  <span class="help-block mt-3"><%= message%> </span>
+<%end%>
+
```

Comment:
@JuanMaGiraldo two blank spaces. Code consistency is really important. Always keep that in mind.

Reply:
Fixed, thanks!